tools/ctl/ctl.c

```diff
@@ -131,7 +134,7 @@ static int read_setup(struct ctl_data *ctl_data)
 	if (ctl_data->no_abi) {
 		header_init(ctl_data);
 		abi_size = sizeof(struct sof_abi_hdr);
-		data_start_int_index += abi_size / sizeof(uint32_t);
+		data_start_int_index += abi_size / sizeof(uint32_t) + BUFFER_ABI_OFFSET;
```

Comment:
Could use here and similar equations for more consistent code the BUFFER_TLV_HEADER_BYTES.